jquery.simulate.js

```diff
@@ -200,8 +200,8 @@ $.extend( $.simulate.prototype, {
 		return event;
 	},
 
-	dispatchEvent: function( elem, type, event ) {
-		if ( elem[ type ] ) {
+	dispatchEvent: function( elem, type, event, options ) {
+		if ( elem[ type ] && !options ) {
```

Comment:
The original bug says this only affects old IE (probably IE8 and lower). Those browsers don't have `dispatchEvent` so that test should probably be first followed by a test for `type === "click" && elem[ type ]` or something similar. Try removing the fix but not the tests from 206bbf873e8f63646fcd1daa3c4713c2ebd66196 and see what fails, that tell you a lot.